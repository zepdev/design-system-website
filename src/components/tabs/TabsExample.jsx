import React, { useState } from "react"
import Tabs from "@zlab-de/zel-react/Tabs"
import Tab from "@zlab-de/zel-react/Tab"
import ZepIconEssentialStar from "@zlab-de/zel-react-icons/ZepIconEssentialStar"
import { makeStyles } from "@material-ui/core/styles"

const useStyles = makeStyles(theme => ({
  container: {
    marginBottom: `${theme.space.l.rem}rem`,
  },
  header: {
    marginBottom: `${theme.space.m.rem}rem`,
  },
  tab: {
    padding: 15,
    background: theme.color.global.lightGray,
  },
}))

function TabsExample() {
  const [value, setValue] = useState(0)

  const classes = useStyles()

  function handleChange(newValue) {
    setValue(newValue)
  }
  return (
    <>
      <div className={classes.container}>
        <p className={classes.header}>Default</p>
        <Tabs value={value} onClick={handleChange} ariaLabel="demo1">
          <Tab label="Tab 1" />
          <Tab label="Tab 2" />
        </Tabs>
        {value === 0 && <p className={classes.tab}>This is Tab 1</p>}
        {value === 1 && <p className={classes.tab}>This is Tab 2</p>}
      </div>
      <div className={classes.container}>
        <p className={classes.header}>Small</p>
        <Tabs
          value={value}
          onClick={handleChange}
          size="small"
          ariaLabel="demo2"
        >
          <Tab label="Tab 1" />
          <Tab label="Tab 2" />
        </Tabs>
        {value === 0 && <p className={classes.tab}>This is Tab 1</p>}
        {value === 1 && <p className={classes.tab}>This is Tab 2</p>}
      </div>
      <div className={classes.container}>
        <p className={classes.header}>Icon</p>
        <Tabs value={value} onClick={handleChange} ariaLabel="demo3">
          <Tab label="Tab 1" icon={<ZepIconEssentialStar />} />
          <Tab label="Tab 2" icon={<ZepIconEssentialStar />} />
        </Tabs>
        {value === 0 && <p className={classes.tab}>This is Tab 1</p>}
        {value === 1 && <p className={classes.tab}>This is Tab 2</p>}
      </div>
      <div className={classes.container}>
        <p className={classes.header}>Icon Small</p>
        <Tabs
          value={value}
          onClick={handleChange}
          size="small"
          ariaLabel="demo4"
        >
          <Tab label="Tab 1" icon={<ZepIconEssentialStar />} />
          <Tab label="Tab 2" icon={<ZepIconEssentialStar />} />
        </Tabs>
        {value === 0 && <p className={classes.tab}>This is Tab 1</p>}
        {value === 1 && <p className={classes.tab}>This is Tab 2</p>}
      </div>
    </>
  )
}

export default TabsExample
