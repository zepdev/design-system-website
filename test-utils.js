import React from "react"
import { render } from "@testing-library/react"
import { ThemeProvider } from "@material-ui/core/styles"
import theme from "./src/gatsby-theme-material-ui-top-layout/theme"

const AllTheProviders = ({ children }) => {
  return <ThemeProvider theme={theme}>{children}</ThemeProvider>
}

const customRender = (ui, options) =>
  render(ui, { wrapper: AllTheProviders, ...options })

// re-export everything
export * from "@testing-library/react"

// override render method
export { customRender as render }
