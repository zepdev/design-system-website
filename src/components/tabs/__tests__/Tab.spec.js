import React from "react"
import { render } from "../../../../test-utils"
import TabsExample from "../TabsExample"

describe("TabsExample", () => {
  it("renders correctly", () => {
    const mockProps = {
      ariaLabel: "test",
    }
    const component = render(<TabsExample {...mockProps} />)
    expect(component).toMatchSnapshot()
  })
})
