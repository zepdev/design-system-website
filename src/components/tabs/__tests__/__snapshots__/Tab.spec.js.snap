// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabsExample renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="makeStyles-container-1"
      >
        <p
          class="makeStyles-header-2"
        >
          Default
        </p>
        <div
          aria-label="demo1"
          class="zep-tabs"
          data-testid="tabs"
          role="tablist"
        >
          <div
            class="zep-tabs__item"
          >
            <button
              aria-selected="true"
              class="zep-tab zep-tab--selected"
              id="tab-0"
              role="tab"
              tabindex="0"
            >
              Tab 1
            </button>
          </div>
          <div
            class="zep-tabs__item"
          >
            <button
              aria-selected="false"
              class="zep-tab"
              id="tab-1"
              role="tab"
              tabindex="0"
            >
              Tab 2
            </button>
          </div>
        </div>
        <p
          class="makeStyles-tab-3"
          data-testid="tabText"
        >
          This is Tab 
          1
        </p>
      </div>
      <div
        class="makeStyles-container-1"
      >
        <p
          class="makeStyles-header-2"
        >
          Small
        </p>
        <div
          aria-label="demo2"
          class="zep-tabs"
          data-testid="tabs"
          role="tablist"
        >
          <div
            class="zep-tabs__item"
          >
            <button
              aria-selected="true"
              class="zep-tab zep-tab--selected zep-tab--small"
              id="tab-0"
              role="tab"
              tabindex="0"
            >
              Tab 1
            </button>
          </div>
          <div
            class="zep-tabs__item"
          >
            <button
              aria-selected="false"
              class="zep-tab zep-tab--small"
              id="tab-1"
              role="tab"
              tabindex="0"
            >
              Tab 2
            </button>
          </div>
        </div>
        <p
          class="makeStyles-tab-3"
        >
          This is Tab 
          1
        </p>
      </div>
      <div
        class="makeStyles-container-1"
      >
        <p
          class="makeStyles-header-2"
        >
          Icon
        </p>
        <div
          aria-label="demo3"
          class="zep-tabs"
          data-testid="tabs"
          role="tablist"
        >
          <div
            class="zep-tabs__item"
          >
            <button
              aria-selected="true"
              class="zep-tab zep-tab--selected zep-tab--icon"
              id="tab-0"
              role="tab"
              tabindex="0"
            >
              <svg
                aria-hidden="true"
                class="zep-tab__icon"
                data-testid="ZepIconEssentialStarIcon"
                focusable="false"
                height="24"
                viewBox="0 0 24 24"
                width="24"
              >
                <path
                  d="M11.85 2.01a1.3 1.3 0 00-1.01.7L8.2 7.94l-5.03.5c-.5.04-.93.37-1.1.85l-.04.13c-.11.44.02.9.35 1.23l4.16 4.12L5 20.36c-.13.5.04 1.02.44 1.35l.11.08c.39.25.88.28 1.3.07L12 19.31l5.16 2.56c.46.22 1 .16 1.4-.16l.1-.1c.33-.32.46-.8.34-1.24l-1.54-5.6 4.16-4.13c.36-.35.48-.88.3-1.36l-.05-.12c-.2-.41-.6-.69-1.06-.73l-5.01-.5-2.63-5.21A1.3 1.3 0 0012 2l-.15.01zM12 3.25c.02 0 .04.02.05.03l2.79 5.53c.1.19.28.32.5.34l5.36.53c.03 0 .04.01.05.03l-.01.05-4.42 4.37a.62.62 0 00-.16.61l1.64 5.95c0 .02 0 .04-.02.05a.05.05 0 01-.06 0l-5.44-2.69a.62.62 0 00-.56 0l-5.43 2.7-.06-.01a.05.05 0 01-.02-.06l1.64-5.94a.62.62 0 00-.17-.6l-4.42-4.4a.05.05 0 01.03-.07l5.38-.54c.21-.02.4-.15.5-.34l2.78-5.52c0-.01.03-.03.05-.03z"
                  fill="currentColor"
                  fill-rule="evenodd"
                  transform="scale(Infinity, Infinity)"
                />
              </svg>
              Tab 1
            </button>
          </div>
          <div
            class="zep-tabs__item"
          >
            <button
              aria-selected="false"
              class="zep-tab zep-tab--icon"
              id="tab-1"
              role="tab"
              tabindex="0"
            >
              <svg
                aria-hidden="true"
                class="zep-tab__icon"
                data-testid="ZepIconEssentialStarIcon"
                focusable="false"
                height="24"
                viewBox="0 0 24 24"
                width="24"
              >
                <path
                  d="M11.85 2.01a1.3 1.3 0 00-1.01.7L8.2 7.94l-5.03.5c-.5.04-.93.37-1.1.85l-.04.13c-.11.44.02.9.35 1.23l4.16 4.12L5 20.36c-.13.5.04 1.02.44 1.35l.11.08c.39.25.88.28 1.3.07L12 19.31l5.16 2.56c.46.22 1 .16 1.4-.16l.1-.1c.33-.32.46-.8.34-1.24l-1.54-5.6 4.16-4.13c.36-.35.48-.88.3-1.36l-.05-.12c-.2-.41-.6-.69-1.06-.73l-5.01-.5-2.63-5.21A1.3 1.3 0 0012 2l-.15.01zM12 3.25c.02 0 .04.02.05.03l2.79 5.53c.1.19.28.32.5.34l5.36.53c.03 0 .04.01.05.03l-.01.05-4.42 4.37a.62.62 0 00-.16.61l1.64 5.95c0 .02 0 .04-.02.05a.05.05 0 01-.06 0l-5.44-2.69a.62.62 0 00-.56 0l-5.43 2.7-.06-.01a.05.05 0 01-.02-.06l1.64-5.94a.62.62 0 00-.17-.6l-4.42-4.4a.05.05 0 01.03-.07l5.38-.54c.21-.02.4-.15.5-.34l2.78-5.52c0-.01.03-.03.05-.03z"
                  fill="currentColor"
                  fill-rule="evenodd"
                  transform="scale(Infinity, Infinity)"
                />
              </svg>
              Tab 2
            </button>
          </div>
        </div>
        <p
          class="makeStyles-tab-3"
        >
          This is Tab 
          1
        </p>
      </div>
      <div
        class="makeStyles-container-1"
      >
        <p
          class="makeStyles-header-2"
        >
          Icon Small
        </p>
        <div
          aria-label="demo4"
          class="zep-tabs"
          data-testid="tabs"
          role="tablist"
        >
          <div
            class="zep-tabs__item"
          >
            <button
              aria-selected="true"
              class="zep-tab zep-tab--selected zep-tab--icon zep-tab--small"
              id="tab-0"
              role="tab"
              tabindex="0"
            >
              <svg
                aria-hidden="true"
                class="zep-tab__icon"
                data-testid="ZepIconEssentialStarIcon"
                focusable="false"
                height="24"
                viewBox="0 0 24 24"
                width="24"
              >
                <path
                  d="M11.85 2.01a1.3 1.3 0 00-1.01.7L8.2 7.94l-5.03.5c-.5.04-.93.37-1.1.85l-.04.13c-.11.44.02.9.35 1.23l4.16 4.12L5 20.36c-.13.5.04 1.02.44 1.35l.11.08c.39.25.88.28 1.3.07L12 19.31l5.16 2.56c.46.22 1 .16 1.4-.16l.1-.1c.33-.32.46-.8.34-1.24l-1.54-5.6 4.16-4.13c.36-.35.48-.88.3-1.36l-.05-.12c-.2-.41-.6-.69-1.06-.73l-5.01-.5-2.63-5.21A1.3 1.3 0 0012 2l-.15.01zM12 3.25c.02 0 .04.02.05.03l2.79 5.53c.1.19.28.32.5.34l5.36.53c.03 0 .04.01.05.03l-.01.05-4.42 4.37a.62.62 0 00-.16.61l1.64 5.95c0 .02 0 .04-.02.05a.05.05 0 01-.06 0l-5.44-2.69a.62.62 0 00-.56 0l-5.43 2.7-.06-.01a.05.05 0 01-.02-.06l1.64-5.94a.62.62 0 00-.17-.6l-4.42-4.4a.05.05 0 01.03-.07l5.38-.54c.21-.02.4-.15.5-.34l2.78-5.52c0-.01.03-.03.05-.03z"
                  fill="currentColor"
                  fill-rule="evenodd"
                  transform="scale(Infinity, Infinity)"
                />
              </svg>
              Tab 1
            </button>
          </div>
          <div
            class="zep-tabs__item"
          >
            <button
              aria-selected="false"
              class="zep-tab zep-tab--icon zep-tab--small"
              id="tab-1"
              role="tab"
              tabindex="0"
            >
              <svg
                aria-hidden="true"
                class="zep-tab__icon"
                data-testid="ZepIconEssentialStarIcon"
                focusable="false"
                height="24"
                viewBox="0 0 24 24"
                width="24"
              >
                <path
                  d="M11.85 2.01a1.3 1.3 0 00-1.01.7L8.2 7.94l-5.03.5c-.5.04-.93.37-1.1.85l-.04.13c-.11.44.02.9.35 1.23l4.16 4.12L5 20.36c-.13.5.04 1.02.44 1.35l.11.08c.39.25.88.28 1.3.07L12 19.31l5.16 2.56c.46.22 1 .16 1.4-.16l.1-.1c.33-.32.46-.8.34-1.24l-1.54-5.6 4.16-4.13c.36-.35.48-.88.3-1.36l-.05-.12c-.2-.41-.6-.69-1.06-.73l-5.01-.5-2.63-5.21A1.3 1.3 0 0012 2l-.15.01zM12 3.25c.02 0 .04.02.05.03l2.79 5.53c.1.19.28.32.5.34l5.36.53c.03 0 .04.01.05.03l-.01.05-4.42 4.37a.62.62 0 00-.16.61l1.64 5.95c0 .02 0 .04-.02.05a.05.05 0 01-.06 0l-5.44-2.69a.62.62 0 00-.56 0l-5.43 2.7-.06-.01a.05.05 0 01-.02-.06l1.64-5.94a.62.62 0 00-.17-.6l-4.42-4.4a.05.05 0 01.03-.07l5.38-.54c.21-.02.4-.15.5-.34l2.78-5.52c0-.01.03-.03.05-.03z"
                  fill="currentColor"
                  fill-rule="evenodd"
                  transform="scale(Infinity, Infinity)"
                />
              </svg>
              Tab 2
            </button>
          </div>
        </div>
        <p
          class="makeStyles-tab-3"
        >
          This is Tab 
          1
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="makeStyles-container-1"
    >
      <p
        class="makeStyles-header-2"
      >
        Default
      </p>
      <div
        aria-label="demo1"
        class="zep-tabs"
        data-testid="tabs"
        role="tablist"
      >
        <div
          class="zep-tabs__item"
        >
          <button
            aria-selected="true"
            class="zep-tab zep-tab--selected"
            id="tab-0"
            role="tab"
            tabindex="0"
          >
            Tab 1
          </button>
        </div>
        <div
          class="zep-tabs__item"
        >
          <button
            aria-selected="false"
            class="zep-tab"
            id="tab-1"
            role="tab"
            tabindex="0"
          >
            Tab 2
          </button>
        </div>
      </div>
      <p
        class="makeStyles-tab-3"
        data-testid="tabText"
      >
        This is Tab 
        1
      </p>
    </div>
    <div
      class="makeStyles-container-1"
    >
      <p
        class="makeStyles-header-2"
      >
        Small
      </p>
      <div
        aria-label="demo2"
        class="zep-tabs"
        data-testid="tabs"
        role="tablist"
      >
        <div
          class="zep-tabs__item"
        >
          <button
            aria-selected="true"
            class="zep-tab zep-tab--selected zep-tab--small"
            id="tab-0"
            role="tab"
            tabindex="0"
          >
            Tab 1
          </button>
        </div>
        <div
          class="zep-tabs__item"
        >
          <button
            aria-selected="false"
            class="zep-tab zep-tab--small"
            id="tab-1"
            role="tab"
            tabindex="0"
          >
            Tab 2
          </button>
        </div>
      </div>
      <p
        class="makeStyles-tab-3"
      >
        This is Tab 
        1
      </p>
    </div>
    <div
      class="makeStyles-container-1"
    >
      <p
        class="makeStyles-header-2"
      >
        Icon
      </p>
      <div
        aria-label="demo3"
        class="zep-tabs"
        data-testid="tabs"
        role="tablist"
      >
        <div
          class="zep-tabs__item"
        >
          <button
            aria-selected="true"
            class="zep-tab zep-tab--selected zep-tab--icon"
            id="tab-0"
            role="tab"
            tabindex="0"
          >
            <svg
              aria-hidden="true"
              class="zep-tab__icon"
              data-testid="ZepIconEssentialStarIcon"
              focusable="false"
              height="24"
              viewBox="0 0 24 24"
              width="24"
            >
              <path
                d="M11.85 2.01a1.3 1.3 0 00-1.01.7L8.2 7.94l-5.03.5c-.5.04-.93.37-1.1.85l-.04.13c-.11.44.02.9.35 1.23l4.16 4.12L5 20.36c-.13.5.04 1.02.44 1.35l.11.08c.39.25.88.28 1.3.07L12 19.31l5.16 2.56c.46.22 1 .16 1.4-.16l.1-.1c.33-.32.46-.8.34-1.24l-1.54-5.6 4.16-4.13c.36-.35.48-.88.3-1.36l-.05-.12c-.2-.41-.6-.69-1.06-.73l-5.01-.5-2.63-5.21A1.3 1.3 0 0012 2l-.15.01zM12 3.25c.02 0 .04.02.05.03l2.79 5.53c.1.19.28.32.5.34l5.36.53c.03 0 .04.01.05.03l-.01.05-4.42 4.37a.62.62 0 00-.16.61l1.64 5.95c0 .02 0 .04-.02.05a.05.05 0 01-.06 0l-5.44-2.69a.62.62 0 00-.56 0l-5.43 2.7-.06-.01a.05.05 0 01-.02-.06l1.64-5.94a.62.62 0 00-.17-.6l-4.42-4.4a.05.05 0 01.03-.07l5.38-.54c.21-.02.4-.15.5-.34l2.78-5.52c0-.01.03-.03.05-.03z"
                fill="currentColor"
                fill-rule="evenodd"
                transform="scale(Infinity, Infinity)"
              />
            </svg>
            Tab 1
          </button>
        </div>
        <div
          class="zep-tabs__item"
        >
          <button
            aria-selected="false"
            class="zep-tab zep-tab--icon"
            id="tab-1"
            role="tab"
            tabindex="0"
          >
            <svg
              aria-hidden="true"
              class="zep-tab__icon"
              data-testid="ZepIconEssentialStarIcon"
              focusable="false"
              height="24"
              viewBox="0 0 24 24"
              width="24"
            >
              <path
                d="M11.85 2.01a1.3 1.3 0 00-1.01.7L8.2 7.94l-5.03.5c-.5.04-.93.37-1.1.85l-.04.13c-.11.44.02.9.35 1.23l4.16 4.12L5 20.36c-.13.5.04 1.02.44 1.35l.11.08c.39.25.88.28 1.3.07L12 19.31l5.16 2.56c.46.22 1 .16 1.4-.16l.1-.1c.33-.32.46-.8.34-1.24l-1.54-5.6 4.16-4.13c.36-.35.48-.88.3-1.36l-.05-.12c-.2-.41-.6-.69-1.06-.73l-5.01-.5-2.63-5.21A1.3 1.3 0 0012 2l-.15.01zM12 3.25c.02 0 .04.02.05.03l2.79 5.53c.1.19.28.32.5.34l5.36.53c.03 0 .04.01.05.03l-.01.05-4.42 4.37a.62.62 0 00-.16.61l1.64 5.95c0 .02 0 .04-.02.05a.05.05 0 01-.06 0l-5.44-2.69a.62.62 0 00-.56 0l-5.43 2.7-.06-.01a.05.05 0 01-.02-.06l1.64-5.94a.62.62 0 00-.17-.6l-4.42-4.4a.05.05 0 01.03-.07l5.38-.54c.21-.02.4-.15.5-.34l2.78-5.52c0-.01.03-.03.05-.03z"
                fill="currentColor"
                fill-rule="evenodd"
                transform="scale(Infinity, Infinity)"
              />
            </svg>
            Tab 2
          </button>
        </div>
      </div>
      <p
        class="makeStyles-tab-3"
      >
        This is Tab 
        1
      </p>
    </div>
    <div
      class="makeStyles-container-1"
    >
      <p
        class="makeStyles-header-2"
      >
        Icon Small
      </p>
      <div
        aria-label="demo4"
        class="zep-tabs"
        data-testid="tabs"
        role="tablist"
      >
        <div
          class="zep-tabs__item"
        >
          <button
            aria-selected="true"
            class="zep-tab zep-tab--selected zep-tab--icon zep-tab--small"
            id="tab-0"
            role="tab"
            tabindex="0"
          >
            <svg
              aria-hidden="true"
              class="zep-tab__icon"
              data-testid="ZepIconEssentialStarIcon"
              focusable="false"
              height="24"
              viewBox="0 0 24 24"
              width="24"
            >
              <path
                d="M11.85 2.01a1.3 1.3 0 00-1.01.7L8.2 7.94l-5.03.5c-.5.04-.93.37-1.1.85l-.04.13c-.11.44.02.9.35 1.23l4.16 4.12L5 20.36c-.13.5.04 1.02.44 1.35l.11.08c.39.25.88.28 1.3.07L12 19.31l5.16 2.56c.46.22 1 .16 1.4-.16l.1-.1c.33-.32.46-.8.34-1.24l-1.54-5.6 4.16-4.13c.36-.35.48-.88.3-1.36l-.05-.12c-.2-.41-.6-.69-1.06-.73l-5.01-.5-2.63-5.21A1.3 1.3 0 0012 2l-.15.01zM12 3.25c.02 0 .04.02.05.03l2.79 5.53c.1.19.28.32.5.34l5.36.53c.03 0 .04.01.05.03l-.01.05-4.42 4.37a.62.62 0 00-.16.61l1.64 5.95c0 .02 0 .04-.02.05a.05.05 0 01-.06 0l-5.44-2.69a.62.62 0 00-.56 0l-5.43 2.7-.06-.01a.05.05 0 01-.02-.06l1.64-5.94a.62.62 0 00-.17-.6l-4.42-4.4a.05.05 0 01.03-.07l5.38-.54c.21-.02.4-.15.5-.34l2.78-5.52c0-.01.03-.03.05-.03z"
                fill="currentColor"
                fill-rule="evenodd"
                transform="scale(Infinity, Infinity)"
              />
            </svg>
            Tab 1
          </button>
        </div>
        <div
          class="zep-tabs__item"
        >
          <button
            aria-selected="false"
            class="zep-tab zep-tab--icon zep-tab--small"
            id="tab-1"
            role="tab"
            tabindex="0"
          >
            <svg
              aria-hidden="true"
              class="zep-tab__icon"
              data-testid="ZepIconEssentialStarIcon"
              focusable="false"
              height="24"
              viewBox="0 0 24 24"
              width="24"
            >
              <path
                d="M11.85 2.01a1.3 1.3 0 00-1.01.7L8.2 7.94l-5.03.5c-.5.04-.93.37-1.1.85l-.04.13c-.11.44.02.9.35 1.23l4.16 4.12L5 20.36c-.13.5.04 1.02.44 1.35l.11.08c.39.25.88.28 1.3.07L12 19.31l5.16 2.56c.46.22 1 .16 1.4-.16l.1-.1c.33-.32.46-.8.34-1.24l-1.54-5.6 4.16-4.13c.36-.35.48-.88.3-1.36l-.05-.12c-.2-.41-.6-.69-1.06-.73l-5.01-.5-2.63-5.21A1.3 1.3 0 0012 2l-.15.01zM12 3.25c.02 0 .04.02.05.03l2.79 5.53c.1.19.28.32.5.34l5.36.53c.03 0 .04.01.05.03l-.01.05-4.42 4.37a.62.62 0 00-.16.61l1.64 5.95c0 .02 0 .04-.02.05a.05.05 0 01-.06 0l-5.44-2.69a.62.62 0 00-.56 0l-5.43 2.7-.06-.01a.05.05 0 01-.02-.06l1.64-5.94a.62.62 0 00-.17-.6l-4.42-4.4a.05.05 0 01.03-.07l5.38-.54c.21-.02.4-.15.5-.34l2.78-5.52c0-.01.03-.03.05-.03z"
                fill="currentColor"
                fill-rule="evenodd"
                transform="scale(Infinity, Infinity)"
              />
            </svg>
            Tab 2
          </button>
        </div>
      </div>
      <p
        class="makeStyles-tab-3"
      >
        This is Tab 
        1
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
